packages/vite-plugin-cloudflare/playground/request-signal/__tests__/signal.spec.ts

```diff
@@ -0,0 +1,12 @@
+import { expect, test, vi } from "vitest";
+import { page } from "../../__test-utils__";
+
+test("renders HTML stream", async () => {
+	const heading = await page.textContent("h1");
+	expect(heading).toBe("Streaming example");
+	const finalChunk = await vi.waitUntil(() => page.textContent("#two"), {
+		timeout: 5000,
+		interval: 500,
+	});
+	expect(finalChunk).toBe("Chunk after 2 seconds");
```

Comment:
Don't we now need to check that the request was cancelled after we received the second chunk?